test/conformance/program/urProgramCreateWithIL.cpp

```diff
@@ -8,6 +8,15 @@
 struct urProgramCreateWithILTest : uur::urContextTest {
     void SetUp() override {
         UUR_RETURN_ON_FATAL_FAILURE(urContextTest::SetUp());
+        // TODO: This should use a query for urProgramCreateWithIL support or
```

Comment:
I guess this falls into the discussion of which entrypoints should be considered optional. Maybe we should create a github issue to update this later?

Reply:
Having an issue for these discussions makes a lot of sense, yes.

Reply:
https://github.com/oneapi-src/unified-runtime/issues/1267